pyvalid/validators/__iterable.py

```diff
@@ -139,13 +159,16 @@ def checkers(self):
     @accepts(object, empty_allowed=bool, element_type=(str, int, float),
              min_val=(int, float), max_val=(int, float))
     def __init__(self, **kwargs):
+        iterable_type = kwargs.get('iterable_type', None)
```

Comment:
Hey @Aparna-BS ,
Can we put it after the `min_val > max_val` check?  Just like it was before. So this code will never be executed in case if `min_val > max_val` .
Thanks!

Reply:
Thanks. Updated the PR as per your comment.

Reply:
Amazing, thanks @Aparna-BS!